<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewFunc</class>
 <widget class="QDialog" name="DialogNewFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="RbSingleTerm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>6</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>单端</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LbFileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>58</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>库文件名：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RbMultiTerm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>6</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>多端</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EbLibFileName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>55</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>29</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>功能名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdFuncName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>27</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LbSPSPic">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>26</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnFuncSymbolSet">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>5</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择默认功能符号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDelFuncSymbol">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删 除</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbFuncType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>功能类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbFuncType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>83</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>接线端口</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>虚拟端口-观测布尔量</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>虚拟端口-观测实数量</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame_Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>505</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="TEditLayout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="LbFuncType_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>T类型名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EbTModelClass">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>111</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
